src/components/TransactionPreview/ActionPreview.tsx

```diff
@@ -1228,6 +1242,20 @@ const ActionPreview = ({
 
     const cardTitle = swap ? 'Asset swap' : 'PLR stake';
 
+    var gasCost;
+    var totalFees = 0;
+    if(swap?.type === 'CROSS_CHAIN_SWAP' && swap?.route) {
```

Comment:
Also reminder to use the auto formatter
```suggestion
    if (swap?.type === 'CROSS_CHAIN_SWAP' && swap?.route) {
```